src/main/java/com/animalrenting/controllers/AnimalList.java

```diff
@@ -0,0 +1,47 @@
+package com.animalrenting.controllers;
+
+import com.animalrenting.models.Animals;
+import com.animalrenting.services.AnimalsServices;
+import org.springframework.boot.context.properties.source.InvalidConfigurationPropertyValueException;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/animals")
+public class AnimalList {
```

Comment:
Controller should have a suffix Controller. Rename AnimalList to AnimalController